apps/explorer/src/app/components/blocks/home/blocks-table.tsx

```diff
@@ -12,31 +12,34 @@ interface BlocksProps {
 
 export const BlocksTable = ({ data, showTransactions }: BlocksProps) => {
   if (!data?.result) {
-    return <>No block data</>;
+    return <div className="mb-28">Awaiting block data</div>;
```

Comment:
chore (blocking): We will render this in an error state as well, should handle errors with feedback